DFE-Digital/sts-plan-technology-for-your-school
# feat: merge contentful environments

## Overview

Addresses ticket [#231955](https://dev.azure.com/dfe-ssp/s190-schools-technology-services/_boards/board/t/STS%202.0%20Scrum%20Teams/Stories/?workitem=231955)

Add JavaScript script to import content from a json file to a specific Contentful environment


## How to review the PR

Include things such as:
- Add a piece of content to the `ContentAndSupport_DevAndTest` Contentful environment
- Export that environment using the `export-processor`
- Add the path to the exported json file to the `content-management` environment variables
- Update the other env variables and set the ENVIRONMENT to `PlanTech_DevAndTest`
- Run the `import-content` script
- Verify that the content has been imported into the `PlanTech_DevAndTest` Contentful environment

## Release requirements

Perhaps use the Contentful merge web app to merge the content types before running the script. This allows a final check of which model changes are going to be made before any imports. In this case, `SKIP_CONTENT_MODEL` can be `true` when running the script.


## Checklist

Delete any rows that do not apply to the PR.
